### PR TITLE
Add sagano testnet configuration

### PR DESCRIPTION
**Pull request by lemastero**
# Description

Currently, build mantis client, do not have a config for sagano network out of the box. You need to rename current conf file.

# Proposed Solution

Copy config testnet-internal-nomad.conf to sagano.conf

# Testing

After building the client you should be able to run:
```bash
./bin/mantis-launcher sagano
```

### CODE REVIEW

File: src/universal/conf/sagano.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+include "base-testnet.conf"
+
+mantis {
```
Comment (enriquerodbe) [inline src/universal/conf/sagano.conf]: Instead of copying everything, wouldn't it be better to `include "testnet-internal-nomad.conf"`?

**Comment by faraggi** [comment]
How about an empty file and importing the previous conf until devops is updated?

File: src/universal/conf/sagano.conf (hunk 1 above)
Comment (lemastero) [reply src/universal/conf/sagano.conf]: Very good idea. Done. Tested locally.

**Review by enriquerodbe — APPROVED** [review]
I saw it later, but I think @faraggi comment was in the same direction

**Review by faraggi — APPROVED** [review]
LGTM

**Pull request merged**